slo_generator/exporters/bigquery.py

```diff
@@ -82,14 +82,13 @@ def create_table(self, project_id, dataset_id, table_id, schema=None):
             table_id (str): Table id to create.
             schema (dict): BigQuery table schema in JSON format.
         """
-        pyschema = []
         if schema is not None:
             schema = TABLE_SCHEMA
-        for row in schema:
-            field = bigquery.SchemaField(row['name'],
+        pyschema = [bigquery.SchemaField(row['name'],
                                          row['type'],
                                          mode=row['mode'])
-            pyschema.append(field)
+                                         for row in schema]
```

Comment:
this is over-indented according to pylint, can you please correct it ?

Reply:
@ocervell done!